pkg/cmd/build/build.go

```diff
@@ -251,7 +251,7 @@ func OptsFromBuildInfo(manifestName, svcName string, b *model.BuildInfo, o *type
 	}
 
 	outputMode := oktetoLog.GetOutputFormat()
-	if o != nil && o.OutputMode != "" {
```

Comment:
I may be in favor of keeping this. We check of `o == nil` at the top of the function but it's a large function. What if that behavior or check changes or is removed? Maybe there should be a unittest for this behavior? What's the linting error?

Reply:
The linting error is `possible nil pointer dereference`. I agree we should have a unit test, but I don't think it will be null at any point

Reply:
If I understand correctly, the problem arises from checking `o != nil`, which expresses that `o` can be `nil` in the other places in this function. The linter suggests 2 outcomes:
- Handle nil
- Suppose that function will not recieve a nil

Reply:
Why it doesn't complain on line `229`? It seems a similar case

```
if o != nil && o.BuildToGlobal {
```

Reply:
Playing around a bit more, seems like the linting is due to 2 things happening in the same scope:
- de referencing a pointer
- checking if the pointer is null

This gives the impression that one of the 2 should be changed:
- if the pointer **cannot** be null, then we **shouldn't** check at any time
- if the pointer **can** be null, we **must** check every time

@ifbyol Seems like the other case is similar but since its in another scope, and in that scope we don't do the 2 things mentioned at the beginning, it might be the case that the pointer could be nil only in that context (inside the if statement)